squid/Dockerfile

```diff
@@ -24,15 +24,14 @@ COPY --from=deps /squid/package.json .
 COPY --from=deps /squid/package-lock.json .
 COPY --from=deps /squid/node_modules node_modules
 COPY --from=builder /squid/lib lib
+COPY .env.example .env
```

Comment:
I'm not sure we should do this, the example file can change and it shouldn't affect the deployment.

Reply:
yup, the `.env.example` should serve as a base for e.g github secrets, and passed in deployment script the way we want.